core/cli/templates/homeTemplate.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {homeViewTemplate, homePageTemplate};
+
+function homeViewTemplate() {
+    let template =
+
+`import React, { Component } from 'react';
+
+class HomeView extends Component {
+
+    render() {
+        return (
+            <div>
+                <h3>Hello from Reframe!</h3>
+            </div>
+        );
+    }
+}
+
+export default HomeView;`;
```

Comment:
Files typically end with a new line

Reply:
Oh really?!  I did not know that.  My OCD bugs me about empty lines at the end of a file, but I'll get those updated ASAP!

Reply:
You can `~ cat path/to/file/without/new/line` and see that the terminal won't start on a new line after the file is printed.